### PR TITLE
Additional animal module test coverage.

### PR DESCRIPTION
**Pull request by michaelRichards99**
This PR moves all existing unit tests for the animal module into its own directory (`test_animal_module`).

It also adds unit tests for `animal_grouping_scenarios.py`, bringing it to 100% coverage.

### CODE REVIEW

File: RUFAS/routines/animal/animal_grouping_scenarios.py
Comment (Isgotkowitz) [inline RUFAS/routines/animal/animal_grouping_scenarios.py:49]: Nice

**Review by Isgotkowitz — APPROVED** [review]
Looks good, nice PR Michael

**Review by allisterakun — APPROVED** [review]

**Comment by KFosterReed** [comment]
@JoeWaddell I was scanning the open PRs and noticed this one that Michael opened before he left. Seems like it got forgotten in his move and the Annual Meeting. Is it still relevant?

**Comment by JoeWaddell** [comment]
> @JoeWaddell I was scanning the open PRs and noticed this one that Michael opened before he left. Seems like it got forgotten in his move and the Annual Meeting. Is it still relevant?

Great find @KFosterReed - I recall seeing this and must have assumed it was merged after approval. I'll fix the merge conflicts and get it merged ASAP.

**Pull request merged**
